projecty/src/main/java/com/service/Service.java

```diff
@@ -71,11 +81,37 @@ public Status sendTweets(String args) throws NullPointerException, TwitterExcept
                 twitterData = new TwitterData(message,twitterHandle,name,profileImageUrl,date);
                 list.add(twitterData);
             }
-
         } catch (TwitterException e) {
             logger.error("Error Occur", e);
-            //  throw new RuntimeException("Run time error");
+             throw new RuntimeException("Run time error");
         }
         return list;
     }
+
+    public List<TwitterData> filterTweet(String filter) throws IllegalArgumentException{
+        List<TwitterData> list= new ArrayList<>();
+        try {
```

Comment:
you can reduce the code ,by calling the getTimeline function.
Apply DRY.

Reply:
hello @vglalitha 
as you told i have done these changes